docker-compose.travis.yml

```diff
@@ -11,13 +11,18 @@ services:
     build:
       context: webapp
       target: build-and-test
+    volumes:
+    #/nitro-web
+       - ./webapp/coverage:/nitro-web/coverage
     environment:
       - GRAPHQL_URI=http://backend:4000
   backend:
     image: humanconnection/nitro-backend:builder
     build:
       context: backend
       target: builder
+    volumes:
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-08T12:13:33Z" title="Wednesday, May 8th 2019, 2:13:33 pm +02:00">May 8, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -11,13 +11,17 @@ services:
     build:
       context: webapp
       target: build-and-test
+    volumes:
+       - coverage:webapp/coverage
     environment:
       - GRAPHQL_URI=http://backend:4000
   backend:
     image: humanconnection/nitro-backend:builder
     build:
       context: backend
       target: builder
+    volumes:
+       - coverage:backend/coverage
```


I'm curious if Travis CI will crash after this. When I tried to add volumes a couple of months ago, Travis could not clean up after this. Reason: The Travis user running the `.travis.yml` is (was?) different from the user of the `docker` daemon. Thus, a file created on the host by `docker-compose` would have a different UNIX file permission. The user on the host machine that tries to clean up after the build would have not enough permissions to remove the created files.